### PR TITLE
Updated ECDbMap for new profile 4.0.0.4

### PR DESCRIPTION
**Pull request by rschili**
ECDbMap will be updated with the next upcoming ECDb release, adding the new ECDbMap schema in this PR

### CODE REVIEW

File: Standard/ECDb/ECDbMap.ecschema.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	- [ForeignKeyConstraint](#foreignkeyconstraint)
+	- [ImportRequiresVersion](#importrequiresversion)
+	- [JoinedTablePerDirectSubclass](#joinedtableperdirectsubclass)
+	- [LinkTableRelationshipMap](#linktablerelationshipmap)
+	- [PropertyMap](#propertymap)
+	- [SchemaMap](#schemamap)
+	- [ShareColumns](#sharecolumns)
+	- [UseRequiresVersion](#userequiresversion)
+
+### **DbIndex** *Sealed* [!badge text="StructClass" kind="info"] [<img src=".././media/imodel-schema-editor-icon.png">](https://imodelschemaeditor.bentley.com/?stage=browse&elementtype=StructClass&id=ECDbMap.DbIndex)
```
Comment (naveedkhan8067) [inline Standard/ECDb/ECDbMap.ecschema.md:30]: I don't think you need to use the `ecjson2md` tool at this point. We use it at part of our build pipeline to automatically convert and generate the relevant docs.

Comment (naveedkhan8067) [reply Standard/ECDb/ECDbMap.ecschema.md:30]: Earlier, I thought you want to run it locally to test the documentation..

Comment (rschili) [reply Standard/ECDb/ECDbMap.ecschema.md:30]: Ah okay I understand. I don't think the documentation mentions this, I just saw every other schema has these files so I thought I had to put them there. It should not hurt to leave them in this PR I guess. But thanks for explaining!

Comment (naveedkhan8067) [reply Standard/ECDb/ECDbMap.ecschema.md:30]: Yes, I think at the moment we don't have a concrete policy for this. On some places it was pushed (mostly older), on others (recent one's) it was not. Probably, the reason is, if your `.remarks.md` file changes then you have to regenerate this `.ecschema.md` file to see the latest changes and one may forget about this.  Our build process automatically does this stuff for us and its output is used in production. On the other hand, the benefit of pushing this file is that we can visually see the output here.

@ColinKerr, should we make a concrete policy for this? or keep it optional?

File: SchemaInventory.json [hunk 2]
```diff
@@ -349,6 +349,20 @@
...
       "dynamic": "No",
       "approved": "Yes"
+    },
+    {
+      "name": "ECDbMap",
+      "path": "Standard\\ECDb\\Released\\ECDbMap.02.00.02.ecschema.xml",
+      "released": true,
+      "version": "02.00.02",
+      "comment": "",
+      "verifier": "",
```
Comment (naveedkhan8067) [inline SchemaInventory.json:359]: Just add a verifier here. Otherwise the PR looks good to me.

Comment (rschili) [reply SchemaInventory.json]: done

File: Standard/ECDb/ECDbMap.ecschema.md (hunk 1 above)
Comment (rschili) [reply Standard/ECDb/ECDbMap.ecschema.md]: I remoed the file after talking to Colin.

Comment (naveedkhan8067) [reply Standard/ECDb/ECDbMap.ecschema.md]: Sounds good!

**Review by naveedkhan8067 — APPROVED** [review]

**Review by ColinKerr — APPROVED** [review]

**Review by ColinKerr — APPROVED** [review]

**Pull request merged**